front/src/modules/favorites/hooks/useFavorites.ts

```diff
@@ -14,6 +17,7 @@ export const useFavorites = () => {
   const [insertCompanyFavoriteMutation] = useInsertCompanyFavoriteMutation();
   const [insertPersonFavoriteMutation] = useInsertPersonFavoriteMutation();
   const [deleteFavoriteMutation] = useDeleteFavoriteMutation();
+  const [updateOneFavoritesMutation] = useUpdateOneFavoriteMutation();
 
   const insertCompanyFavorite = (companyId: string) => {
```

Comment:
I am surprised that we don't need to compute the favorite position in this call while inserting the new favorite

Reply:
ok sure. Both insertCompanyFavorite and insertPersonFavorite right?

Reply:
I think so!